eslint.config.mjs

```diff
@@ -0,0 +1,5 @@
+import eslintConfig from "@pnpm/eslint-config";
+
+export default [{
+    ignores: ["**/fixtures", "**/__fixtures__", ".meta-updater/src/index.ts"],
```

Comment:
meta-updater wasn't ignored before these changes.

Reply:
Not sure why but it is ignored for me on the main branch 🤷‍♂️ Changing this file contents doesn't throw any eslint errors (on my machine, maybe my setup is wrong).

I can remove it from the ignore list though if you want.

Reply:
Running the following command:

```sh
pnpm eslint .meta-updater/src/index.ts
```

Outputs (before and after this PR changes):

```
pnpm/.meta-updater/src/index.ts
  0:0  warning  File ignored by default.  Use a negated ignore pattern (like "--ignore-pattern '!<relative/path/to/filename>'") to override

✖ 1 problem (0 errors, 1 warning)
```

Reply:
it should be included. Also, your new ts files don't follow the conventions used by the repo.

Reply:
fyi it wasn't included in the ts config. Fixed.

Could you be more specific? What can I change or which conventions?